### PR TITLE
Docs: What's new in 7.2 edits

### PR DESCRIPTION
**Pull request by oddlittlebird**
Edited content, reordered a few items so the document would flow more logically, and added links to updated content.

If the Transformations feature is coming out of beta, we should explicitly state that in this document as well.

### CODE REVIEW

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 1]
```diff
@@ -28,19 +27,16 @@ The main highlights are:
   - [**Organization-wide report settings**]({{< relref "#organization-wide-report-settings" >}})
   - [**Report grid layout**]({{< relref "#report-grid-layout" >}})
 
-## Date formatting options
+## New date formatting options added
```
Comment (torkelo) [inline docs/sources/guides/whats-new-in-v7-2.md:30]: if you update a heading you need to update the link to this section as well (at the top of the article we list an overview with links to each section)

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 2]
```diff
@@ -53,58 +49,68 @@ interval_month = YYYY-MM
...
+There is also experimental support to use the browser location and language to dynamically change the current date format for each user. This feature is disabled by default.
+
+The [Configuration]({{< relref "../administration/configuration.md#date_formats" >}}) topic has been updated as a result of this change.
+
+## Field options are out of beta!
+
+After lots of testing and user feedback, we removed the beta label from the configuration options in the Field and Override tabs. This release also includes the following feature enhancements.
 
-## Table column filters
+### Added table column filters
```
Comment (torkelo) [inline docs/sources/guides/whats-new-in-v7-2.md:60]: this also needs updated link

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 3]
```diff
@@ -53,58 +49,68 @@ interval_month = YYYY-MM
...
 
-The new column filter option allows you to dynamically apply value filters to any column. This option is disabled but
-can be enabled for all or a specific column using an override rule.
+You can now dynamically apply value filters to any table column. This option is can be enabled for all columns or one specific column using an override rule.
 
 {{< docs-imagebox img="/img/docs/v72/table_column_filters.png" max-width="800px" caption="Table column filters" >}}
 
-## New and improved transforms
+### New field override selection options
```
Comment (torkelo) [inline docs/sources/guides/whats-new-in-v7-2.md:66]: needs updated link at the top of the article

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 4]
```diff
@@ -53,58 +49,68 @@ interval_month = YYYY-MM
...
 
-Grafana 7.2 includes a new group by transform that allows you to group by multiple fields and add any number of aggregations for other fields.
+You can now add override rules that use a regex matcher to choose which fields to apply rules to.
 
-There is also an update to the labels to fields transform that allow you to pick one label and use that as the name of the value field.
+The [Field configuration options]({{< relref "../panels/field-configuration-options.md" >}}) content has been updated as a result of these changes.
```
Comment (torkelo) [inline docs/sources/guides/whats-new-in-v7-2.md:70]:
```suggestion
The [Field options]({{< relref "../panels/field-options.md" >}}) content has been updated as a result of these changes.
```

Renamed this page to Field options as we agreed in the docs channel :) https://github.com/grafana/grafana/pull/27604

**Review by torkelo — COMMENTED** [review]
Great updates! Just need to update the links in the top section that links to the sections

File: docs/sources/guides/whats-new-in-v7-2.md [hunk 5]
```diff
@@ -53,58 +49,68 @@ interval_month = YYYY-MM
...
+- The [Labels to fields]({{< relref "../panels/transformations.md#labels-to-fields">}}) transformation now allows you to pick one label and use that as the name of the value field.
+- You can now move transformations up and down. Remember that transformations are processed in the order they are listed in the UI, so think before you move something!
 
-You can now add override rules that use a regex matcher and overrides that match on field type.
+{{< docs-imagebox img="/img/docs/v72/transformations.gif" max-width="800px" caption="Group by and reordering of transformations" >}}
 
 ## Sensitive alert notification channel settings are now stored encrypted in the database
 
-Before Grafana v7.2 alert notification channels did not store sensitive settings/secrets such as API tokens and password encrypted in the database. In Grafana v7.2, creating a new alert notification channel will store sensitive settings encrypted in the database.
+Alert notification channels now store sensitive settings and secrets, such as API tokens and passwords, encrypted in the database.
```
Comment (marefr) [inline docs/sources/guides/whats-new-in-v7-2.md:84]: Looks good

File: docs/sources/guides/whats-new-in-v7-2.md (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/guides/whats-new-in-v7-2.md:30]: Good catch, thank you!

**Review by torkelo — APPROVED** [review]
Looks good!

**Pull request merged**